app/models/person.rb

```diff
@@ -56,6 +56,10 @@ def biography
     details["body"]
   end
 
+  def translations
+    available_translations.each { |t| t[:text] = language_name(t[:locale]) }
```

Comment:
I found this a bit confusing to read, it might be better to use `each_with_object`.

```suggestion
    available_translations.each_with_object do |translation, memo|
      memo[:locale] = translation[:locale]
      memo[:base_path] = translation[:base_path
      memo[:text] = language_name(translation[:locale])
    end
```

I prefer this because it's explicit as well, we can be sure we'll have the right fields in the dictionary.

Reply:
Also, it would be good to support the `:active` key as well as per https://components.publishing.service.gov.uk/component-guide/translation-nav so the current locale isn't clickable.

Reply:
I thought it would be more "ruby" to keep it short but your suggestion does make it clearer! 👍 

I'll look into the getting the `active` thing working